setup/helpers/perf_helper.rb

```diff
@@ -668,7 +572,9 @@ def setup_gatling_proxy
     end
     step 'install rvm, bundler' do
       on metric, 'gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3'
-      on metric, 'curl -sSL https://get.rvm.io | bash -s stable --ruby=2.2.5'
+      on metric, 'curl -sSL https://get.rvm.io | bash -s stable'
+      # Work around install issue in Centos7: https://github.com/rvm/rvm/issues/4234
+      on metric, 'rvm install 2.2.5 --disable-binary'
```

Comment:
if this is to work around known 2.2.5 rvm issues in centos, this change should be in a separate commit.  oh, i see the comment now 🤣 
but yeah, separate commit for ease of revert later

Reply:
i guess install also implicitly uses that latest version?  or maybe if there is only one version?
the man page is vague on it.